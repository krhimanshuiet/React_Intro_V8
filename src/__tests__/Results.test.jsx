import {expect,test} from "vitest"
import {render} from "@testing-library/react"
import Result from "./../Results"

test("renders correct with no pets",async () => {
    const  {asFragment} = render(<Result pets={[]}/>)
    expect(asFragment()).toMatchSnapshot();
})