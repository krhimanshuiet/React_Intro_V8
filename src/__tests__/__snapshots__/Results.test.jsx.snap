// Vitest Snapshot v1

exports[`renders correct with no pets 1`] = `
<DocumentFragment>
  <div
    class="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-3"
  >
    <h1>
      No Pets Found
    </h1>
  </div>
</DocumentFragment>
`;
